src/provider/cloud-element-templates/create/TemplateElementFactory.js

```diff
@@ -62,7 +63,7 @@ export default class TemplateElementFactory {
       throw new Error('template is invalid');
     }
 
-    const type = appliesTo[0];
+    const type = (elementType && elementType.value) || appliesTo[0];
```

Comment:
I just see we could have made [`value` required for the `elementType` binding](https://github.com/camunda/element-templates-json-schema/blob/master/packages/element-templates-json-schema-shared/src/defs/base.json#L53). 

Maybe a good follow-up.

Reply:
Follow-up PR in the element templates: https://github.com/camunda/element-templates-json-schema/pull/53

Reply:
However, as `elementType` itself could still be `undefined`, we still need the check at this position